Example/Pods/Local Podspecs/TSSwiftMarkdownParser.podspec.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "TSSwiftMarkdownParser",
```

Comment:
Great question. I just did that cause the code was based off of TSMarkdownParser.  But I have changed it a lot.  Time for a rebrand!